src/AudioIO.cpp

```diff
@@ -1463,7 +1463,7 @@ void AudioIO::StartMonitoring( const AudioIOStartStreamOptions &options )
    if (!success) {
       auto msg = XO("Error opening recording device.\nError code: %s")
          .Format( Get()->LastPaErrorString() );
-      ShowErrorDialog( FindProjectFrame( mOwningProject ),
+      ShowExceptionDialog( FindProjectFrame( mOwningProject ),
```

Comment:
This might fire every time somebody simply has a loose cable connection... would the collected data be valuable?

Reply:
There is only one way to find out :-)